.github/workflows/events-nightly.yml

```
@@ -92,7 +116,9 @@ jobs:
         include:
           # the config used in '.azure-pipelines/gpu-tests.yml'
           - {python_version: "3.7", pytorch_version: "1.8"}
```

@Borda Could you update this for #12373? Even though we have (3.9, 1.10) below, I'd like to use (3.7, 1.10) in the GPU CI since py3.7 is still used in kaggle/colab notebooks.
```suggestion
          - {python_version: "3.7", pytorch_version: "1.10"}
```